packages/fxa-settings/src/lib/glean/index.ts

```diff
@@ -358,6 +358,18 @@ const recordEventMetric = (
     case 'account_pref_recovery_key_submit':
       accountPref.recoveryKeySubmit.record();
       break;
+    case 'account_pref_two_step_auth_submit':
```

Comment:
Maybe just another strange diff, but shouldn't there only be two additional case statements needed added for this PR?

Reply:
There are only two new ones, I just grouped the account_pref cases together (moved up `two_step_auth_submit` and `change_password_submit`)